src/components/content/heroes/aurora/weather-easter-egg/sun-percentage-calculator.ts

```diff
@@ -0,0 +1,21 @@
+export const getSunlightPercentage = (
+  sunriseTime: number,
+  sunsetTime: number,
+) => {
+  const time = +new Date();
```

Comment:
If your function accepts a parameter `time` you would gain a truly pure function that you can test without mocking the system time in your specs as you can control what to pass in. That's not crucial but something I absolutely recommend to change to make your code more understandable and easier to test 🙏

```
getSunlightPercentage(dateA, dateB, currentTime)
```